src/acceptance-test/java/org/zalando/nakadi/webservice/hila/SubscriptionAT.java

```diff
@@ -390,6 +392,52 @@ public void whenStatsOnNotInitializedSubscriptionThanCorrectResponse() throws IO
         }
     }
 
+    @Test
+    public void whenLightStatsOnNotInitializedSubscriptionThenCorrectResponse() throws IOException {
+        final String et = createEventType().getName();
+        final Subscription s = createSubscriptionForEventType(et);
+        final Response response = when().get("/subscriptions?show_status=true").thenReturn();
```

Comment:
it's an acceptance test, what if there will be other subscriptions created by other tests? I think it's better to differ the subscription you created (e.g. by owning_application)